content/en/admin/migrating.md

```diff
@@ -15,17 +15,19 @@ This guide was written with Ubuntu Server in mind; your mileage may vary for oth
 
 ## Basic steps {#basic-steps}
 
-1. Set up a new Mastodon server using the [Production Guide]({{< relref "install" >}}) (however, don’t run `mastodon:setup`).
-2. Stop Mastodon on the old server (e.g. `systemctl stop 'mastodon-*.service'`).
+1. Set up a new Mastodon server using the [Production Guide]({{< relref "install" >}}) (however, don’t run `mastodon:setup` and only leave the PostgreSQL service running).
+2. Stop Mastodon and Redis on the old server (e.g. `systemctl stop 'mastodon-*.service'` and `systemctl stop redis-server.service).
 3. Dump and load the PostgreSQL database using the instructions below.
 4. Copy the `system/` files using the instructions below. (Note: if you’re using S3, you can skip this step.)
 5. Copy the `.env.production` file.
-6. Run `RAILS_ENV=production bundle exec rails assets:precompile` to compile Mastodon
-7. Run `RAILS_ENV=production ./bin/tootctl feeds build` to rebuild the home timelines for each user.
+6. Copy the Redis database from `/var/lib/redis/` to the new server.
+7. Run `RAILS_ENV=production bundle exec rails assets:precompile` to compile Mastodon
 8. Start Mastodon on the new server.
-9. Update your DNS settings to point to the new server.
-10. Update or copy your Nginx configuration, and re-run LetsEncrypt as necessary.
-11. Enjoy your new server!
+9. Run `RAILS_ENV=production ./bin/tootctl feeds build` to rebuild the home timelines for each user.
```

Comment:
Redis needs to be up for this.

Reply:
Correct, I had that information in the "detailed steps" but forgot to include that in the "basic steps". I also put the wrong order for starting the services which I'm fixing now.